lib/author.rb

```diff
@@ -0,0 +1,13 @@
+class Author
+  attr_reader :name, :books
+  def initialize(name)
+    @name = name[:first_name] + ' ' + name[:last_name]
+    @books = []
+  end
+
+  def write(book_name, book_publication)
+    new_book = Book.new({author_first_name: @name.split[0], author_last_name: @name.split[1], title: book_name, publication_date: book_publication})
```

Comment:
Love the use of `split` here - you could also initialize with `@first_name` and `@last_name`, just for use inside the class.